src/Action/ProfileDashboardAction.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\UserBundle\Action;
+
+use Sonata\UserBundle\Model\UserInterface;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Security\Core\Authentication\Token\Storage\TokenStorageInterface;
+use Twig\Environment;
+
+final class ProfileDashboardAction
```

Comment:
After thinking about this for a few days. I would move this to an other bundle, e.g. ProfileBundle. The UserBundle should only contain some basic user management stuff.

Reply:
I thinking about it too. It is simmilar to CustomerBundle profile where profile part is require and used(order list, customer data...). Here frontend does not exist, so it is not require. ProfileBundle is good idea, but it is to small for seperate bundle. Then i think about tutorial.

What do you think about create section "users tutorial" where anyone can share own solution?

Reply:
> [...] ProfileBundle is good idea, but it is to small for seperate bundle. Then i think about tutorial.

What's the problem about a small bundle? It's easier to maintain and fulfils a special use case, which is a good idea.

> What do you think about create section "users tutorial" where anyone can share own solution?

The problem about a tutorial is that you have to copy & paste many parts and you won't get informed about updates :(

Reply:
Btw I've created such a small bundle which is reusable: https://github.com/nucleos/NucleosProfileBundle

Reply:
> The problem about a tutorial is that you have to copy & paste many parts and you won't get informed about updates :(

Lets say users have to create bundle with docs and tests (mybe even with special description file). Then just add link this to tutorial site. Import script should check is bundle is correct, docs and test. Aslo can check update automatically.

> What's the problem about a small bundle? It's easier to maintain and fulfils a special use case, which is a good idea.

We can do it. Some change like multiple menus, editable dashboards will be nice to have.

Reply:
This sounds like a new make:user-profile  command